### PR TITLE
Add roadmap

### PR DESCRIPTION
**Pull request by caiopiccirillo**

### CODE REVIEW

**Comment by caiopiccirillo** [comment]
@LLagu, this is a draft for roadmap. If you see something wrong or you'd like to add something, just let me know.

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+- [ ] Implement more advanced features for controlling joints, such as speed and precision.
+- [ ] Add support for multiple joints in a single model.
+- [ ] Add support for nested joints.
+- [ ] Control spawn camera position and projection.
+
+# Phase 4 (v0.4.0):
+
+- [ ] Add support for more complex interactions between joints.
+- [ ] Add support for physics simulation of joints.
```
Comment (LLagu) [inline ROADMAP.md:27]: I am a bit confused by this. Isn't Bevy already taking care of the physics and isn't the joint physics partially working in the sample you showed me?

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- [ ] Add support for networking, so that multiple users can control the joints in a shared scene.
+
+# Phase 5 (v0.5.0):
+
+- [ ] Add additional features and polish, such as a user interface, documentation, and tests.
```
Comment (LLagu) [inline ROADMAP.md:32]: Shall we take care of the documentation afterwards or add a skeleton that we can expand as we progress?

File: ROADMAP.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+# Phase 6 (v1.0.0):
+
+- [ ] Release the first stable version of the software.
+
+<!-- 
+Error handling: How will you handle errors that occur while loading or controlling the digital twins?
```
Comment (LLagu) [inline ROADMAP.md:39]: Could you please explain the purpose of these four questions? :sweat_smile: Are those points of concern?

File: ROADMAP.md (hunk 1 above)
Comment (caiopiccirillo) [reply ROADMAP.md:27]: You're completely right, I forgot to delete this step in this phase.

File: ROADMAP.md (hunk 3 above)
Comment (caiopiccirillo) [reply ROADMAP.md:39]: Yes, they really are some concerns that we might think more about it in the future. I just left it there as reminder 😅

**Review by LLagu — APPROVED** [review]

**Pull request merged**